hydrosdk/servable.py

```diff
@@ -7,32 +7,30 @@
 from .exceptions import ServableException
 from .image import DockerImage
 from .model import Model
+from .predictor import Predictable
 
 
 class ServableStatus(Enum):
     """
     # TODO add more information about statuses
-    Servable can be in one of four states:
-        1. STARTING -
-        2. SERVING -
-        3. NOT_SERVING -
-        4. NOT_AVAILABLE -
+    Servable can be in one of four states
     """
     STARTING = 2
+    UNKNOWN = 2
     SERVING = 3
     NOT_SERVING = 0
     NOT_AVAILABLE = 1
 
 
-class Servable:
+class Servable(Predictable):
     """
     Servable is an instance of a model version which could be used in application or by itself as it exposes various endpoints to your model version: HTTP, gRPC, and Kafka.
     (https://hydrosphere.io/serving-docs/latest/overview/concepts.html#servable)
     """
     BASE_URL = "/api/v2/servable"
 
     @staticmethod
-    def model_version_json_to_servable(mv_json: dict, cluster):
+    def model_version_json_to_servable(mv_json: dict, cluster, grpc_cluster=None):
```

Comment:
❌ what is `grpc_cluster` and why `cluster` is not enough?

Reply:
The main purpose of `Cluster` class is to wrap all network and transport.

By introducing the dupe of it in domain classes, we create a precedent which blurs the responsibilities of `Cluster`.

If you want to cover a situation where http and grpc endpoints have different addresses, it's better to put an additional field and logic inside `Cluster`.

Reply:
Do we have any activity on this comment? Any ideas about how this can be improved? @KineticCookie @trthhrtz

Reply:
Yeah, as mentioned above
> If you want to cover a situation where http and grpc endpoints have different addresses, it's better to put an additional field and logic inside Cluster.

Reply:
@KineticCookie @Valenzione 
done